### PR TITLE
Use shortcode for all warning blocks

### PR DESCRIPTION
**Pull request by pierwill**

### CODE REVIEW

File: content/enterprise_influxdb/v1.2/features/cluster-commands.md [hunk 1]
```diff
@@ -460,8 +460,8 @@ The command aborts the `copy-shard` command that was copying shard `39` from `cl
 Removes a meta node and/or data node from the cluster.
 Use `leave` instead of the [`remove-meta`](#remove-meta) and [`remove-data`](#remove-data) arguments if you set up your InfluxEnterprise cluster with the [QuickStart Installation](/enterprise_influxdb/v1.2/quickstart_installation/cluster_installation/) process.
 
-<dt>The `leave` argument is destructive; it erases all metastore information from meta nodes and all data from data nodes.
-Use `leave` only if you want to *permanently* remove a node from a cluster.</dt>
+{{% warn %}}The `leave` argument is destructive; it erases all metastore information from meta nodes and all data from data nodes.
+Use `leave` only if you want to *permanently* remove a node from a cluster.{{% /warn %}}
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.2/features/cluster-commands.md]: These may need a newline after the shortcode to render correctly. I'm not sure.

File: content/enterprise_influxdb/v1.2/features/cluster-commands.md [hunk 2]
```diff
@@ -536,8 +536,8 @@ The system doesn't remove a data node from the cluster because it doesn't find a
 Removes a data node from a cluster.
 Use `remove-data` instead of the [`leave`](#leave) argument if you set up your InfluxEnterprise cluster with the [Production Installation](/enterprise_influxdb/v1.2/production_installation/) process.
 
-<dt>The `remove-data` argument is destructive; it erases all data from the specified data node.
-Use `remove-data` only if you want to *permanently* remove a data node from a cluster.</dt>
+{{% warn %}}The `remove-data` argument is destructive; it erases all data from the specified data node.
+Use `remove-data` only if you want to *permanently* remove a data node from a cluster.{{% /warn %}}
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.2/features/cluster-commands.md]: Same comment about newline. Double check and make sure these render correctly.

File: content/enterprise_influxdb/v1.2/features/cluster-commands.md [hunk 3]
```diff
@@ -564,8 +564,8 @@ The command removes a data node running at `cluster-data-node-03:8088` from an e
 Removes a meta node from the cluster.
 Use `remove-meta` instead of the [`leave`](#leave) argument if you set up your InfluxEnterprise cluster with the [Production Installation](/enterprise_influxdb/v1.2/production_installation/) process.
 
-<dt>The `remove-meta` argument is destructive; it erases all metastore information from the specified meta node.
-Use `remove-meta` only if you want to *permanently* remove a meta node from a cluster.</dt>
+{{% warn %}}The `remove-meta` argument is destructive; it erases all metastore information from the specified meta node.
+Use `remove-meta` only if you want to *permanently* remove a meta node from a cluster.{{% /warn %}}
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.2/features/cluster-commands.md]: Same

**Comment by pierwill** [comment]
- [x] Tests pass (no build errors)

**Pull request merged**